### ISSUE TITLE
Villagers don't claim empty beds

### ISSUE THREAD

**Issue by malv1x** [Luke100000/minecraft-comes-alive#1037]
**Describe the bug**
My village has enough beds, they are registered in the blueprint, but my villagers claim they are all occupied though the homes have no names assigned. Even when I manually assigned everyone a home (which was possible when I broke and then placed down the beds again) they unclaimed them and were complaining about a lack of beds a few minutes later.

**To Reproduce**
Lead the villagers to an unoccupied house. 
Click on Actions - Locations - Home
They say it's nice but all beds are occupied.
This happens even when I tell them to find a home on their own.

**Version**
MCA Version 7.5.21
Minecraft version 1.20.1
Modloader - Fabric 0.16.9 through the Curseforge Windows app

**Latest.log**
[latest.log](https://github.com/user-attachments/files/17965444/latest.log)

**Comment by Luke100000** [comment]
Are you sure they are actually homeless and didn't already claim said bed?

**Comment by malv1x** [comment]
Yep, definitely sure. I have 16 villagers and 16 beds, and not all of them claimed, so I know they are homeless.

**Comment by Luke100000** [comment]
So what does the villager say when you ask him to go home?

**Comment by malv1x** [comment]
There are no more empty beds or i don't have a home

**Comment by Luke100000** [comment]
Then I ran out of ideas. Please find a minimal way to reconstruct this issue.

**Comment by malv1x** [comment]
![Screenshot 2024-12-01 133147](https://github.com/user-attachments/assets/4240449b-279d-4ae4-994c-8ec26c7f3c2a)
![Screenshot 2024-12-01 133215](https://github.com/user-attachments/assets/aa357275-ec6a-45b9-94ef-0477d6c64f1e)

**Comment by malv1x** [comment]
Only two people in a four-bed house, Lodovico still claiming they are all occupied

![Screenshot 2024-12-01 133215](https://github.com/user-attachments/assets/66aa927a-36b7-46c5-9a13-5c4b2c630422)
![Screenshot 2024-12-01 133147](https://github.com/user-attachments/assets/4625ac22-ac90-4782-8df9-e8a633b0d60f)

**Comment by Luke100000** [comment]
A minimal example is a setup with just MCA, on e,g, superflat, and instructions on how to repeat this reliable
